lib/rubocop.rb

```diff
@@ -743,7 +744,11 @@
 
 # rubocop:enable Style/RequireOrder
 
-unless File.exist?("#{__dir__}/../rubocop.gemspec") # Check if we are a gem
+set_rubocop_required_features =
+  ENV['__RUBOCOP_ASSUME_WE_ARE_NOT_A_GEM'] == 'true' ||
+  !File.exist?("#{__dir__}/../rubocop.gemspec") # Check if we are a gem
+
+if set_rubocop_required_features
```

Comment:
I'm not sure we actually need this variable given it's used only once and its name is slightly misleading.